docs/conf.py

```diff
@@ -199,42 +209,50 @@
 github_issues_url = 'https://github.com/sunpy/sunpy/issues/'
 
 # -- Options for the Sphinx gallery -------------------------------------------
-
-if on_rtd and os.environ.get('READTHEDOCS_PROJECT').lower() != 'sunpy':
```

Comment:
can you merge this into gallery try block at the top?

Reply:
K